assets/js/modules/analytics/datastore/report.js

```diff
@@ -54,8 +56,9 @@ const fetchGetReportStore = createFetchStore( {
 		invariant( isPlainObject( options ), 'Options for Analytics report must be an object.' );
 		invariant( isValidDateRange( options ), 'Either date range or start/end dates must be provided for Analytics report.' );
 
-		const { metrics, dimensions, orderby } = options;
+		const { metrics, dimensions, orderby } = normalizeReportOptions( options );
 
+		invariant( metrics.length, 'Requests must specify at least one metric for an Analytics report.' );
```

Comment:
@felixarntz this is normalized first now so we can rely on `metrics` being an array of strings or objects for this length check.